test/AutoRest.TestServer.Tests/body-complex.cs

```diff
@@ -274,18 +270,17 @@ public void CheckComplexPrimitiveDouble()
         public Task GetComplexPrimitiveDateTimeRfc1123() => Test(async (host, pipeline) =>
         {
             var result = await PrimitiveOperations.GetDateTimeRfc1123Async(ClientDiagnostics, pipeline, host);
-            Assert.AreEqual(DateTimeOffset.Parse("Mon, 01 Jan 0001 12:00:00 GMT"), result.Value.Field);
+            Assert.AreEqual(DateTimeOffset.Parse("Mon, 01 Jan 0001 00:00:00 GMT"), result.Value.Field);
```

Comment:
I had the actual value wrong initially because I used the description value, not the one from the actual testserver script.